pkg/ddl/backfilling_scheduler.go

```diff
@@ -329,18 +331,37 @@ type ingestBackfillScheduler struct {
 
 func newIngestBackfillScheduler(ctx context.Context, info *reorgInfo,
 	sessPool *sess.Pool, tbl table.PhysicalTable, distribute bool) *ingestBackfillScheduler {
+	avgRowSize := estimateAvgRowSize(tbl)
 	return &ingestBackfillScheduler{
 		ctx:        ctx,
 		reorgInfo:  info,
 		sessPool:   sessPool,
 		tbl:        tbl,
+		avgRowSize: avgRowSize,
 		taskCh:     make(chan *reorgBackfillTask, backfillTaskChanSize),
 		resultCh:   make(chan *backfillResult, backfillTaskChanSize),
 		poolErr:    make(chan error),
 		distribute: distribute,
 	}
 }
 
+func estimateAvgRowSize(tbl table.PhysicalTable) int {
+	size := 0
+	for _, col := range tbl.Meta().Columns {
+		ft := col.FieldType
+		storeLen := ft.StorageLength()
+		if storeLen != types.VarStorageLen {
+			size += storeLen
+		} else {
+			if types.IsString(ft.GetType()) {
```

Comment:
will the Json and blob be treated as char type？What is the return length of them.